tags/tags-impl/impl/src/resources/xmlsamples/mesh.xml

```diff
@@ -0,0 +1,4816 @@
+     <Term  ConceptPreferredTermYN="N"  IsPermutedTermYN="N"  LexicalTag="NON"  RecordPreferredTermYN="N">
+      <TermUI>T000014</TermUI>
+      <String>Injuries, Abdominal</String>
+      <DateCreated>
+       <Year>1986</Year>
+       <Month>01</Month>
+       <Day>01</Day>
+      </DateCreated>
+      <EntryVersion>INJ ABDOMINAL</EntryVersion>
+      <ThesaurusIDlist>
+       <ThesaurusID>UNK (19XX)</ThesaurusID>
+      </ThesaurusIDlist>
+     </Term>
+     <Term  ConceptPreferredTermYN="N"  IsPermutedTermYN="Y"  LexicalTag="NON"  RecordPreferredTermYN="N">
+      <TermUI>T000014</TermUI>
+      <String>Abdominal Injury</String>
+     </Term>
+     <Term  ConceptPreferredTermYN="N"  IsPermutedTermYN="Y"  LexicalTag="NON"  RecordPreferredTermYN="N">
+      <TermUI>T000014</TermUI>
+      <String>Injury, Abdominal</String>
+     </Term>
+    </TermList>
+   </Concept>
+  </ConceptList>
+ </DescriptorRecord>
+<DescriptorRecord DescriptorClass = "1">
+  <DescriptorUI>D000008</DescriptorUI>
+  <DescriptorName>
+   <String>Abdominal Neoplasms</String>
+  </DescriptorName>
+  <DateCreated>
+   <Year>1999</Year>
+   <Month>01</Month>
+   <Day>01</Day>
+  </DateCreated>
+  <DateRevised>
+   <Year>1995</Year>
+   <Month>06</Month>
+   <Day>08</Day>
+  </DateRevised>
+  <AllowableQualifiersList>
+   <AllowableQualifier>
+    <QualifierReferredTo>
+     <QualifierUI>Q000469</QualifierUI>
+      <QualifierName>
+      <String>parasitology</String>
+      </QualifierName>
+    </QualifierReferredTo>
+    <Abbreviation>PS</Abbreviation>
+   </AllowableQualifier>
+   <AllowableQualifier>
+    <QualifierReferredTo>
+     <QualifierUI>Q000276</QualifierUI>
+      <QualifierName>
+      <String>immunology</String>
+      </QualifierName>
+    </QualifierReferredTo>
+    <Abbreviation>IM</Abbreviation>
+   </AllowableQualifier>
+   <AllowableQualifier>
+    <QualifierReferredTo>
+     <QualifierUI>Q000382</QualifierUI>
+      <QualifierName>
+      <String>microbiology</String>
+      </QualifierName>
+    </QualifierReferredTo>
+    <Abbreviation>MI</Abbreviation>
+   </AllowableQualifier>
+   <AllowableQualifier>
+    <QualifierReferredTo>
+     <QualifierUI>Q000175</QualifierUI>
+      <QualifierName>
+      <String>diagnosis</String>
+      </QualifierName>
+    </QualifierReferredTo>
+    <Abbreviation>DI</Abbreviation>
+   </AllowableQualifier>
+   <AllowableQualifier>
+    <QualifierReferredTo>
+     <QualifierUI>Q000503</QualifierUI>
+      <QualifierName>
+      <String>physiopathology</String>
+      </QualifierName>
+    </QualifierReferredTo>
+    <Abbreviation>PP</Abbreviation>
+   </AllowableQualifier>
+   <AllowableQualifier>
+    <QualifierReferredTo>
+     <QualifierUI>Q000530</QualifierUI>
+      <QualifierName>
+      <String>radiography</String>
+      </QualifierName>
+    </QualifierReferredTo>
+    <Abbreviation>RA</Abbreviation>
+   </AllowableQualifier>
+   <AllowableQualifier>
+    <QualifierReferredTo>
+     <QualifierUI>Q000531</Qua
```

Comment:
These are sample files to show the format needed in the files that can be used to import tags. These files can go to the tomcat/sakai/tags/ and they will be used by the jobs to load/update the tags in the collections. 

I've a manual to explain all that that I will add in a readme file.